dns/route53/require.tf

```diff
@@ -0,0 +1,9 @@
+# Terraform version and plugin versions
+
+terraform {
+  required_version = ">= 0.12.0"
+}
+
+provider "aws" {
+  version = "2.31.0"
```

Comment:
Please use `required_providers` syntax to avoid instantiating the provider. See https://www.terraform.io/docs/configuration/modules.html#provider-version-constraints-in-modules for more details.

Reply:
Ok, done!